### DISCUSSION TITLE
6月14日 付款后邮箱没收到相关信息 请解决或退款

### DISCUSSION THREAD

**Discussion by szyww2** [getlantern/lantern-forum-cn#574] [General]
邮箱是：850342890@qq.com
订单号：2024061422001391121401312176
![22334](https://github.com/getlantern/lantern-forum-cn/assets/172788050/a8a2cbff-dc23-4bb4-871c-fa41ebdc0e7b)

**Comment by MaxMKMax** [comment]
@szyww2 您好. 我帮您提交问题和安排.

**Comment by szyww2** [comment]
请给我激活码 发到我的邮箱



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年6月15日 00:44
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: szyww2 ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 6月14日 付款后邮箱没收到相关信息 请解决或退款 (Discussion #574)





 
@szyww2 您好. 我帮您提交问题和安排.

**Comment by szyww2** [comment]
我还是没有收到任何邮件 能尽快解决吗



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年6月15日 00:44
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: szyww2 ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 6月14日 付款后邮箱没收到相关信息 请解决或退款 (Discussion #574)





 
@szyww2 您好. 我帮您提交问题和安排.

**Comment by MaxMKMax** [comment]
@szyww2 您好. 这是您专属的激活码:
KDK3K-WRJR4-2KQDY-JKR9D-VBQJ9
